tox.ini

```diff
@@ -29,6 +32,10 @@ commands =
     python setup.py test -q
 deps = {[testenv]deps}
 
+[testenv:jython]
```

Comment:
This stanza is not needed: tox already knows how to invoke jython (even if it can't build the virtualenv ATM).